ecs/context_test.go

```diff
@@ -43,7 +43,7 @@ func TestCreateContextDataFromEnv(t *testing.T) {
 	assert.Equal(t, desc, "credentials read from environment")
 }
 
-func TestCreateContextDataByKeys(t *testing.T) {
+func _TestCreateContextDataByKeys(t *testing.T) {
```

Comment:
I guess this is a temporary ignore, not to commit in the PR ?

Reply:
Ops... Fixed